conda/signature.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import base64
+import hashlib
+from os.path import abspath, expanduser, join
+
+from conda.compat import PY3
+
+from Crypto.PublicKey import RSA
+
+
+KEYS = {}
+KEYS_DIR = abspath(expanduser('~/.conda/keys'))
+
+
+def sig2ascii(i):
```

Comment:
Please write a docstring that defines, very accurately, what is a valid value for "i" here.  I would also suggest that the parameter should be called "sig" and not "i".